lib/components/app/locale-selector.tsx

```diff
@@ -65,20 +46,20 @@ const LocaleSelector = (props: LocaleSelectorProps): JSX.Element => {
     <NavDropdown
       id="locale-selector"
       pullRight
-      title={<CurrentLocaleFlag style={{ width: 15 }} />}
+      title={<Icon style={{ color: '#fff' }} type="globe" />}
```

Comment:
Super nit, but could we make the color `color: rgba(255, 255, 255, 0.85);`? This lets some of the bg bleed through (while still being accessible!) and matches the view switcher